boards/arm/lpcxpresso11u68/support/lpc_checksum.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2013 Bas Stottelaar
```

Comment:
@MaureenHelm is it OK to have this Python script here?

Reply:
@henrikbrixandersen @MaureenHelm @carlescufi We added the README file in boards/arm/lpcxpresso11u68/support needed by the new license process.

Reply:
lpc_checksum.py has been added as a dependency in the script/requirements-extra.txt file.